resources/src/books.php

```diff
@@ -129,6 +129,24 @@ public function issue($bid, $id)
       if ($row2 = $stmt2->fetch(PDO::FETCH_ASSOC)) {
          if ($row2["returned"] == "0") {
             echo "Book already issued to you!";
+         } else {
+            $sql1 = "SELECT available FROM Book WHERE Book.id='" . $bid . "'";
+            $stmt1 = $this->pdo->query($sql1);
+            $row1 = $stmt1->fetch(PDO::FETCH_ASSOC);
+            if ((int)$row1["available"] > 0) {
+               $q = ((int)$row1["available"]) - 1;
+               $sql = 'UPDATE Book SET available=? where Book.id=?';
+               $stmt = $this->pdo->prepare($sql);
+               if ($stmt->execute([$q, $bid]) === TRUE) {
+                  $sql2 = 'INSERT into book_user (bid,id,issued_on) values ("' . $bid . '","' . $id . '","' . date('y-m-d') . '")';
+                  if ($this->pdo->query($sql2) === TRUE) {
+                     // header("LOCATION: userdash.php");
+                     echo "Issued";
+                  }
+               }
+            } else {
+               echo "Sorry the book isn't available";
+            }
```

Comment:
Is this part of this PR?

Reply:
yes .

Reply:
@guptahemant  this is part probably fixes some "book issue" function

Reply:
@JayKandari  please review em